stingray/tests/test_powerspectrum.py

```diff
@@ -33,6 +33,7 @@ def test_make_empty_periodogram(self):
         assert ps.norm == "frac"
```

Comment:
We should definitely test somewhere that the values of the errors are correct as well